### DISCUSSION TITLE
5ghz Wifi Required or Recommended?

### DISCUSSION THREAD

**Discussion by Tahan365** [LizardByte/community#435] [Sunshine] [answered]
Reading through the minimum requirements for Sunshine, I noticed that both Host and Client need connection to 5ghz Wifi.

I’d like to know if that’s absolutely necessary, since I’ve been trying to set up and use my PS Vita (which is 2.4ghz only) with Moonlight to connect to Nvidia Shield/Game Stream.  It works fine when connected to the same network, but I’ve not been able to get it to work with different networks (and I can’t find 1. anything that says separate networks doesn’t work, and 2. how to make it work)

When stumbling upon Sunshine, I thought my Novidia struggles were over.  But if 5ghz is the only option, I guess it’s back to the drawing board.

**Comment by ReenigneArcher** [comment, marked as answer]
It might work with a lower performance network, but you're more likely to face performance issues.

Regarding streaming over the internet there are guides out there. I think the moonlight wiki has one.
